### PR TITLE
Code to delete the wdb cluster and the "old" rdb cluster

### PR DESCRIPTION
**Pull request by David2464**
-Created an endofperiod function which runs if in `period mode. In the wdb, this will delete the cluster if it is the "old" process by checking if upd is set to insert. Also ensures there is a new rdb up before it is deleted.

-Created an endofperiod function which runs if in `period mode. In the rdb, will delete the cluser if it is the "old" process and if there are no queries awaiting execution by checking with the gateway.

### CODE REVIEW

File: code/rdb/endofperiod.q [hunk 1]
```diff
@@ -6,15 +6,15 @@ rolloverendofperiod:{[currp;nextp;data]
 	/-Create a list of start times of rdb's,m including current process so that list is not empty
 	times:.proc.starttimeUTC , @[;".proc.starttimeUTC";()]each h;
 	/-If we are the new process, exit function, do not want to close handle
```
Comment (zanbuchanan) [inline code/rdb/endofperiod.q:8]: This comment doesn't match what the code is doing

**Review by zanbuchanan — APPROVED** [review]

File: code/rdb/endofperiod.q [hunk 2]
```diff
@@ -5,16 +5,18 @@ rolloverendofperiod:{[currp;nextp;data]
 	h:exec w from .servers.SERVERS where proctype=`rdb,not w=0N;
 	/-Create a list of start times of rdb's,m including current process so that list is not empty
 	times:.proc.starttimeUTC , @[;".proc.starttimeUTC";()]each h;
-	/-If we are the new process, exit function, do not want to close handle
+	/-Extract gateway handles of RDB processes
```
Comment (jonathonmcmurray) [inline code/rdb/endofperiod.q:8]: This comment is confusing IMO - I think it's clearer as

```suggestion
	/-Extract gateway handles
```

File: docs/12-healthcheck.md
Comment (jonathonmcmurray) [inline docs/12-healthcheck.md:1]: is this change relevant to the PR? make sure only relevant changes are included please

File: env.q
Comment (jonathonmcmurray) [inline env.q:1]: similar to other comment, are these changes relevant to PR?

**Review by jonathonmcmurray — APPROVED** [review]
